src/components/header/gist-modal/renderer.tsx

```diff
@@ -417,7 +417,7 @@ class GistModal extends React.PureComponent<Props, State> {
               )
             ) : (
               <span>
-                Login with{' '}
+                Login with {/* eslint-disable-next-line react/jsx-no-target-blank */}
                 <a href={`${BACKEND_URL}auth/github`} target="_blank">
```

Comment:
Let's use `/* eslint-disable-line react/jsx-no-target-blank */` on the actual line.

Reply:
@domoritz is this what you mean?
```
{/* eslint-disable-next-line react/jsx-no-target-blank */} <a href={`${BACKEND_URL}auth/github`} target="_blank"> 
```
ESLint automatically lints it to the following
```
Login with {/* eslint-disable-next-line react/jsx-no-target-blank */}
<a href={`${BACKEND_URL}auth/github`} target="_blank">
```

Reply:
Disable lint for the actual line, not the next line.

Reply:
@domoritz So I used `// eslint-disable-line react/jsx-no-target-blank`.
```
<span>
  Login with{' '}
  <a href={`${BACKEND_URL}auth/github`} target="_blank"> {/* eslint-disable-line react/jsx-no-target-blank */}
    GitHub
  </a>{' '}
  to see all of your personal gist.
</span>
```
But gets transformed to:
```
<span>
  Login with{' '}
  <a href={`${BACKEND_URL}auth/github`} target="_blank">
    {' '}
    {/* eslint-disable-line react/jsx-no-target-blank */}
    GitHub
  </a>{' '}
  to see all of your personal gist.
</span>
```
and fails `yarn lint`.

Reply:
Ok

Reply:
How about you extract the link into a variable then.

Reply:
No, it doesn't work

Reply:
```
/* eslint-disable */
<a href={LOGIN_URL} target="_blank">
/* eslint-enable */
```
How about this?

Reply:
I'll play with it and merge.